puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-hammer-nightly-systest.yaml

```diff
@@ -0,0 +1,35 @@
+- job:
+    name: foreman-hammer-nightly-systest
+    node: 'el && ipv6'
+    project-type: matrix
+    logrotate:
+      daysToKeep: 8
+      maxToKeep: 30
+    scm:
+      - forklift
+    triggers:
+      - timed: 'H 5 * * 1,4'
+    axes:
+      - axis:
+          type: user-defined
+          name: os
+          values:
+            - centos7
+    builders:
+      - systest:
+          repo: nightly
+          repo_environment: release
```

Comment:
Shouldn't this be staging

Reply:
Agreed.

Reply:
@ekohl @ehelms this one is also currently set to `release`.